packages/myst-cli/src/config.ts

```diff
@@ -275,12 +275,12 @@ export async function resolveToAbsolute(
 ) {
   let message: string | undefined;
   if (opts?.allowRemote && isUrl(relativePath)) {
-    const cacheFilename = `config-item-${computeHash(relativePath)}${extname(relativePath)}`;
+    const cacheFilename = `config-item-${computeHash(relativePath)}${extname(new URL(relativePath).pathname)}`;
```

Comment:
This allows query params, e.g. `logo: https://github.com/rowanc1/pics/blob/main/grapes-wide.png?raw=true`. However, the extensions still needs to be in the URL. (We could optimize slightly by getting extension from content type header...)